crates/matrix-sdk-indexeddb/src/cryptostore.rs

```diff
@@ -1051,3 +1091,21 @@ mod tests {
     }
     cryptostore_integration_tests! { integration }
 }
+
+// FIXME: the tests pass, if run one by one, but run all together locally,
+//        as well as CI fails...
```

Comment:
IndexedDB encrypted crypto-tests fails when run all at once

We temporarily had to disable the `encrypted_tests` in the indexeddb crypto store, because running them one at a time works, but alltogether fails locally as well as on the CI with:

```
wasm-pack test --firefox --headless -- --no-default-features --features e2e-encryption -- matrix_sdk_indexeddb::cryptostore::encrypted_tests::integration
[INFO]: Checking for the Wasm target...
    Finished dev [unoptimized + debuginfo] target(s) in 0.12s
[INFO]: Installing wasm-bindgen...
    Finished test [unoptimized + debuginfo] target(s) in 0.12s
     Running unittests (/home/ben/Dev/matrix/matrix-rust-sdk/target/wasm32-unknown-unknown/debug/deps/matrix_sdk_indexeddb-89792c546a510b2e.wasm)
Set timeout to 20 seconds...
Running headless tests in Firefox on `http://127.0.0.1:40105/`
Try find `webdriver.json` for configure browser's capabilities:
Not found
Failed to detect test as having been run. It might have timed out.
output div contained:
    running 58 tests
    
    test matrix_sdk_indexeddb::cryptostore::encrypted_tests::integration::save_account_via_generic_save ... ok
    test matrix_sdk_indexeddb::cryptostore::encrypted_tests::integration::save_account ... ok
    test matrix_sdk_indexeddb::cryptostore::encrypted_tests::integration::load_account ... ok

driver status: signal: 9                          
driver stdout:
    1652274475706       geckodriver     INFO    Listening on 127.0.0.1:40105
    1652274475886       mozrunner::runner       INFO    Running command: "/usr/bin/firefox" "--marionette" "-headless" "-no-remote" "-profile" "/tmp/rust_mozprofilerJ0jf0"
    1652274476200       Marionette      INFO    Marionette enabled
    [GFX1-]: RenderCompositorSWGL failed mapping default framebuffer, no dt
    console.warn: SearchSettings: "get: No settings file exists, new profile?" (new NotFoundError("Could not open the file at /tmp/rust_mozprofilerJ0jf0/search.json.mozlz4", (void 0)))
    1652274476982       Marionette      INFO    Listening on port 33109
    Read port: 33109
    1652274511361       Marionette      INFO    Stopped listening on port 33109

driver stderr:
    *** You are running in headless mode.
    
    ###!!! [Child][PContentChild] Error: RunMessage(msgname=PContent::Reply_DiscardBrowsingContext) Channel closing: too late to send/recv, messages will be lost
    
    
    ###!!! [Parent][PContentParent] Error: Send(msgname=PContent::Msg_DestroyBrowsingContextGroup) Closed channel: cannot send/recv
    
    JavaScript error: resource://gre/modules/Sqlite.jsm, line 1105: Error: Sqlite.jsm has been shutdown. Cannot open connection to: /tmp/rust_mozprofilerJ0jf0/protections.sqlite
    
    ###!!! [Parent][PImageBridgeParent] Error: RunMessage(msgname=PImageBridge::Msg_WillClose) Channel closing: too late to send/recv, messages will be lost
    

Error: some tests failed                          
error: test failed, to rerun pass '--lib'
Error: Running Wasm tests with wasm-bindgen-test failed
Caused by: failed to execute `cargo test`: exited with exit status: 1
```